spec/src/init.makam

```diff
@@ -0,0 +1,79 @@
+% A specification of the data description language.
```

Comment:
For files like these, you might prefer to use the literate Markdown-based syntax for Makam. You would have to name the file `init.md`, uncomment the documentation, and enclose actual makam code in \`\`\`makam-code blocks.
You would then have to change the `repl` command to `makam src/init.md -`.

Reply:
I probably would prefer some docs that could be compiled from Makam - with an index and cross-links between symbols. Kind of because this is more something that is intended to be consumed non-linearly, than in the form of a top-down document like is assumed with literate programming.

But yeah, that said, I realise that would be a bunch of work to implement! Doing it with markdown might be a good interim solution because then I could make the specification a subdirectory of a [Docusaurus](https://v2.docusaurus.io/) site, for instance.